scripts/ci/run_perftest.sh

```diff
@@ -33,9 +33,7 @@ cat << EOF > /etc/telegraf/telegraf.conf
   json_time_key = "time"
   json_time_format = "unix"
   tag_keys = [
-    "i_type",
```

Comment:
These will show up as fields now right? Based on my limited telegraf experience they should and that should be good...just wanting to double check vs. what you've seen.

Reply:
Ah I see that almost the exact same question was asked on the PR this is backported from, nevermind!